spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/PropertiesConverter.java

```diff
@@ -5,11 +5,30 @@
 
 package com.datastrato.gravitino.spark.connector;
 
+import com.datastrato.gravitino.spark.connector.hive.HivePropertyConstants;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 /** Transform table properties between Gravitino and Spark. */
 public interface PropertiesConverter {
   Map<String, String> toGravitinoTableProperties(Map<String, String> properties);
 
   Map<String, String> toSparkTableProperties(Map<String, String> properties);
+
+  /** Remove 'option.' from property key name. */
+  static Map<String, String> transformOptionProperties(Map<String, String> properties) {
+    return properties.entrySet().stream()
+        .collect(
+            Collectors.toMap(
+                entry -> {
+                  String key = entry.getKey();
+                  if (key.startsWith(HivePropertyConstants.SPARK_OPTION_PREFIX)) {
+                    return key.substring(HivePropertyConstants.SPARK_OPTION_PREFIX.length());
+                  } else {
+                    return key;
+                  }
+                },
```

Comment:
Are there any issues or documentation about this?

Reply:
I didn't see the document, I'll try to find it.

Reply:
I'm afraid there's no document, spark add `option.` to `option properties` and `SERDEPROPERTIES`,  please refer to https://github.com/apache/spark/pull/28026